src/wp-admin/about.php

```diff
@@ -101,143 +118,177 @@
 			</div>
 		</div>
 
 		<div class="about__section has-2-columns is-wider-left">
-			<div class="column about__image is-vertically-aligned-center">
-				<img src="https://s.w.org/images/core/5.8/about-duotone.png" alt="" />
+			<div class="column about__image is-vertically-aligned-center is-edge-to-edge">
+				<img src="https://make.wordpress.org/core/files/2021/12/about-59-list-view.png" alt="" />
 			</div>
 			<div class="column">
 				<h3>
-					<?php _e( 'Style and Colorize Images' ); ?>
+					<?php _e( 'A revamped List View' ); ?>
 				</h3>
 				<p>
-					<?php
-					_e( 'Colorize your image and cover blocks with duotone filters! Duotone can add a pop of color to your designs and style your images (or videos in the cover block) to integrate well with your themes. You can think of the duotone effect as a black and white filter, but instead of the shadows being black and the highlights being white, you pick your own colors for the shadows and highlights. There’s more to learn about how it works in the documentation.' );
-					?>
+					<?php _e( 'In 5.9, the List View lets you drag and drop your content exactly where you want it. Managing complex documents is easier, too: simple controls let you expand and collapse sections as you build your site &#8212; and add HTML anchors to your blocks to help users get around the page.' ); ?>
+				</p>
+			</div>
+		</div>
+
+		<div class="about__section has-2-columns is-wider-right">
+			<div class="column">
+				<h3>
+					<?php _e( 'A better Gallery block' ); ?>
+				</h3>
+				<p>
+					<?php _e( 'Treat every image in a Gallery block the same way you would treat it in the Image block.' ); ?>
+				</p>
+				<p>
+					<?php _e( 'Style every image in your gallery differently from the next (with different crops, or duotones, for instance) or make them all the same. And change the layout with drag-and-drop.' ); ?>
 				</p>
 			</div>
+			<div class="column about__image is-vertically-aligned-center is-edge-to-edge">
+				<img src="https://make.wordpress.org/core/files/2021/12/about-59-gallery-block.png" alt="" />
+			</div>
 		</div>
 
 		<hr class="is-large" />
 
 		<div class="about__section">
 			<h2 class="aligncenter" style="margin-bottom:0;">
-				<?php _e( 'For Developers to Explore' ); ?>
+				<?php
+				printf(
+					/* translators: %s: Version number. */
+					__( 'WordPress %s for developers' ),
+					$display_version
+				);
+				?>
 			</h2>
-			<div class="column about__image is-vertically-aligned-center">
-				<picture>
-					<source srcset="https://s.w.org/images/core/5.8/about-theme-json.png, https://s.w.org/images/core/5.8/about-theme-json-2x.png 2x" />
-					<img src="https://s.w.org/images/core/5.8/about-theme-json.png" alt="" />
-				</picture>
-			</div>
 		</div>
 
-		<div class="about__section has-1-column">
+		<div class="about__section has-2-columns">
 			<div class="column">
 				<h3>
-					<?php _e( 'Theme.json' ); ?>
+					<?php _e( 'Introducing Block Themes' ); ?>
```

Comment:
Should "Block" and "Themes" be capitalized for consistency with other h3s?

Reply:
[Chatting with Mary Baum](https://wordpress.slack.com/archives/C02RQBWTW/p1641315016053000?thread_ts=1641314689.051900&cid=C02RQBWTW), she notes:
>Should be sentence case like the others. Great catch!

```suggestion
					<?php _e( 'Introducing block themes' ); ?>
```